modules/ipi-install-configuring-nodes.adoc

```diff
@@ -87,3 +87,33 @@ To enable Secure Boot manually, refer to the hardware guide for the node and exe
 ====
 Red Hat does not support Secure Boot with self-generated keys.
 ====
+
+[discrete]
+== Configuring the Compatibility Support Module for Fujitsu iRMC
+
+The Compatibility Support Module (CSM) configuration provides the support for legacy BIOS backward compatibility with UEFI systems.
+
+[NOTE]
+====
+When you are deploying with iRMC, you must configure the CSM, otherwise the cluster installation might fail.
+====
+
+[IMPORTANT]
+====
+Before you configure the CSM, ensure that you have disabled the `Secure Boot Control`, which is available under `Security` -> `Secure Boot Configuration`.
+====
+
+To configure the CSM, refer to the hardware guide for the node and execute the following:
+
+. Boot the node and select the BIOS menu.
+. Under `Advanced` tab, select `CSM Configuration` from the list.
+. Enable the `Launch CSM` and set the following values:
```

Comment:
If "`Launch CSM`" is a reference to a UI element, you could change the back-ticks to bold font.